doc/conf.py

```diff
@@ -40,6 +42,7 @@
     'sphinx.ext.mathjax',
     'sphinx.ext.githubpages',
     'sphinx.ext.autosummary',
+    'sphinxcontrib.tikz',
     'numpydoc'
```

Comment:
So it's numpydoc. Please stick to the docstring conventions for `numpydoc` to make the most out of it